docs/apigateway.md

```diff
@@ -0,0 +1,110 @@
+# Exposing your Actions via API Gateway (Experimental)
+
+In OpenWhisk you can invoke your actions using the [REST API](./reference.md#rest-api) using an HTTP Request only via a __POST__ method.
+This requires the HTTP client to make the request using the OpenWhisk authorization API key which is a master
+key that in addition of allowing to invoke an action, also allows for deleting and creating more actions.
+
+This experimental feature will allow you to invoke an action with HTTP methods other than POST, and not requiring to use 
+your OpenWhisk authorization API key from your HTTP client making the request.
```

Comment:
Slight rewording for clarity.  `This experimental feature will allow you to invoke an action with HTTP methods other than POST and without the action's authorization API key.`